src/filerepository.js

```diff
@@ -265,12 +283,13 @@ class FileLoader {
 		this.id = uid();
 
 		/**
-		 * A `File` instance associated with this file loader.
+		 * Additional wrapper over a file promise passed to this loader.
```

Comment:
The constructor of `FileLoader` has outdated docs for `@param {File}` which should be `@param {Promise.<File>}`.

Reply:
I cannot comment the code that wasn't changed hence the location of this comment.

Reply:
Also the param name to be updated also

Reply:
👍